postgresql-17.yaml

```diff
@@ -407,6 +417,96 @@ subpackages:
       pipeline:
         - uses: test/tw/ldd-check
 
+  # This subpackage essentially rebuilds postgres with python and should therefore be the very last package built!!!
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}
+    description: python${{range.key}} version of ${{vars.pypi-package}}
+    dependencies:
+      provider-priority: ${{range.value}}
+    pipeline:
+      - runs: make distclean
+      - runs: |
+          PYTHON=python${{range.key}}  PYTHON3=python${{range.key}}  ./configure \
+            --host=${{host.triplet.gnu}} \
+            --build=${{host.triplet.gnu}} \
+            --prefix=/usr \
+            --bindir=/usr/libexec/${{vars.mangled-package-name}} \
+            --datadir=/usr/share/${{vars.mangled-package-name}} \
+            --docdir=/usr/share/doc/${{vars.mangled-package-name}} \
+            --mandir=/usr/share/${{vars.mangled-package-name}}/man \
+            --with-openssl \
+            --with-libedit-preferred \
+            --with-uuid=e2fs \
+            --with-libxml \
+            --with-libxslt \
+            --with-icu \
+            --with-lz4 \
+            --with-zstd \
+            --with-gssapi \
+            --with-ldap \
+            --with-llvm \
+            --with-python \
+            --enable-thread-safety
+      - uses: autoconf/make
```

Comment:
I wonder if there's a more specific target we could call to avoid building *everything* 5 times.